WalletWasabi.Fluent/Views/SearchBar/SearchBar.axaml.cs

```diff
@@ -1,15 +1,45 @@
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia;
 using Avalonia.Controls;
+using Avalonia.Controls.ApplicationLifetimes;
 using Avalonia.Markup.Xaml;
+using ReactiveUI;
+using WalletWasabi.Fluent.ViewModels.SearchBar;
 
 namespace WalletWasabi.Fluent.Views.SearchBar;
 
 public class SearchBar : UserControl
 {
+	private readonly CompositeDisposable _disposables = new();
+
 	public SearchBar()
 	{
 		InitializeComponent();
 	}
 
+	protected override void OnAttachedToVisualTree(VisualTreeAttachmentEventArgs e)
+	{
+		MessageBus.Current
+			.Listen<CloseSearchBarMessage>()
+			.Do(_ => FocusMainWindow())
+			.Subscribe()
+			.DisposeWith(_disposables);
```

Comment:
nACK. Messages are the first sign of violating MVVM rules, they are makes the code absolutely hard to read. We should never use them.